Bottom: ab5cd8fb89fadbe6a9676051ca4e8e19faf38739
Top:    20b98a674cc90fb74869e152c8a4056f4566b144
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 17:55:32 +0100

Refresh of w-c-split-gathering-routines-f

---

diff --git a/src/draw.c b/src/draw.c
index e163c9a..08ea7e4 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1518,7 +1518,7 @@ DrawElementPackage (ElementTypePtr Element)
   END_LOOP;
   ARC_LOOP (Element);
   {
-    _drar_arc (arc);
+    _draw_arc (arc);
   }
   END_LOOP;
 }
